it/utils/ETMPWiremock.scala

```diff
@@ -438,16 +438,16 @@ trait ETMPWiremock {
           .withStatus(status)))
```

Comment:
Please remove mockResponseForStubETMPPayload and mockResponseForETMPPayload and etmpPayloadAsJson